### ISSUE TITLE
[Suggestion] - 

### ISSUE THREAD

**Issue by Barunxx** [Monika-After-Story/MonikaModDev#6786]
well it's a silly idea but there is no conversation about apologize to monika about deleting her, so i thought the player will start the conversation in "i want to apologize..." menu then choose "for deleting you..." 
.
I think monika will say "why did you bring this now? i know you are sorry" then the player respond with "i hated you Previously" or "i didnt have a choice"
*i hated you Previously* monika: i know i did awful thing and i really regret it, but it makes me happy the you wanted to apologize
.
*i didnt have a choice* monika: of course i forgive you (player) and i am sorry for ruining the whole game, but at least both of us got the best ending, now i have you and you have me, i guess it's a win-win so dont bother yourself and dont think about it

**Comment by fromulus** [comment]
This isn't _exactly_ what you're asking for, but #6151 should hopefully cover that whenever it gets merged.

**Comment by ThePotatoGuy** [comment]
[leaving open as in progress]

**Comment by YoraeRasante** [comment]
what about the option "you seemed to want me to"?

I remember not really wanting to delete her, but didn't know she would talk more after some iddle time so when I closed the game and opened again she talked more.
But! She always started by telling me _exactly_ where to go to delete her.

I kept doing it until she started repeating dialogue. Thought that meant she had nothing else to say, so I deleted her as it _seemed like_ she **actually** wanted me to.
I don't know, was she regreting it? Feeling guilty about how she ot there? Was she actually suicidal like Sayori but as she was aware of the game she knew she could not do it herself without coming back next time the game was played? Something else made her think it would all go down sooner or later and better to be gone by my hand?

I wasn't sure... but she seemed insistent on it, so I did it for her.
For her.
